packages/frontend/src/components/SettingsValidator/index.tsx

```diff
@@ -47,7 +48,9 @@ export const SettingsValidator: FC<{ projectUuid: string }> = ({
                 >
                     <Text fw={500} fz="xs" c="gray.6">
                         {!!data?.length
-                            ? `Last validated at: ${data[0].createdAt}`
+                            ? `Last validated at: ${moment(
+                                  data[0].createdAt,
+                              ).format('MMMM Do YYYY, h:mm:ss a')}`
```

Comment:
there is a formatTime function in schedulers view which you can either duplicate or reuse for consistency
```
export const formatTime = (date: Date) =>
    dayjs(date).format('YYYY/MM/DD hh:mm A');
```

Reply:
Sure, will make this change